<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GameWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>571</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="fieldGridLayout">
     <item row="3" column="0">
      <widget class="QLabel" name="Field3_0">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>ressources/FieldIcon.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="Field0_0">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>ressources/FieldIcon.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="Field3_2">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>ressources/FieldIcon.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="Field1_1">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>ressources/FieldIcon.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="Field2_3">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>ressources/FieldIcon.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="Field0_2">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>ressources/FieldIcon.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="Field2_2">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>ressources/FieldIcon.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Field1_0">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>ressources/FieldIcon.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="Field3_3">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>ressources/FieldIcon.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="Field1_3">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>ressources/FieldIcon.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="Field0_1">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>ressources/FieldIcon.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="Field2_1">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>ressources/FieldIcon.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="Field1_2">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>ressources/FieldIcon.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="Field2_0">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>ressources/FieldIcon.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="Field3_4">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>ressources/FieldIcon.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="Field1_4">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>ressources/FieldIcon.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="Field2_4">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>ressources/FieldIcon.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="Field3_1">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>ressources/FieldIcon.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="Field0_3">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>ressources/FieldIcon.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="Field0_4">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>ressources/FieldIcon.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="Field4_0">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>ressources/FieldIcon.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="Field4_1">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>ressources/FieldIcon.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="Field4_2">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>ressources/FieldIcon.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="Field4_3">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>ressources/FieldIcon.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QLabel" name="Field4_4">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>ressources/FieldIcon.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>610</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PLAY</string>
    </property>
   </widget>
   <widget class="QComboBox" name="obstaclesComboBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>610</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="pointsLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	background-color: white
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
